styles.css

```diff
@@ -618,3 +618,125 @@ body {
   line-height: 24px;
   padding: 12px;
 }
+
+footer {
+  padding-top: 7.115625em;
+  background-color: rgba(255, 255, 255, 1);
+  font-family: "poppins", sans-serif;
+}
+
+#contact-form {
+  max-height: 779px;
+  border-top-left-radius: 79px;
+  background: rgba(96, 112, 255, 1);
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+#contact-form h5 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  font-weight: 700;
+  font-size: 40px;
+  color: rgba(255, 255, 255, 1);
+  line-height: 52px;
+  padding: 7.93512rem 1.5rem 0.75rem 0;
+  margin: 0;
+}
+
+#contact-form p {
+  font-style: normal;
+  font-size: 20px;
+  font-weight: 400;
+  color: rgba(255, 255, 255, 1);
+  text-align: center;
+  padding: 0 1.5rem 3.5rem;
+  margin: 0;
+}
+
+.input {
+  display: flex;
+  width: 25.2191rem;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: stretch;
+  padding: 0 1.5rem;
+}
+
+.input-item {
+  margin-bottom: 1rem;
+  height: 3rem;
+  border-radius: 0.5rem;
+  border-style: none;
+}
+
+.input textarea {
+  margin-bottom: 1rem;
+  height: 10.625rem;
+  border-radius: 0.5rem;
+  border-style: none;
+}
+
+.input button {
+  width: 8.75rem;
+  height: 3rem;
+  border-style: none;
+  border-radius: 0.5rem;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  color: #6070ff;
+  margin-bottom: 2.8125rem;
+  padding-bottom: 0;
+}
```

Comment:
- [ ] [REQUIRED] Let's ensure that the button interactions such as hover, enabled, disabled, and enabled, pressed are styled as shown in the Figma template1.
### An Iage of Template 1
![Screenshot from 2022-06-08 12-31-34](https://user-images.githubusercontent.com/58692590/172606291-0e964e5c-d060-4ef1-9b5e-e01f46a99b7b.png)